core/signeddata.go

```diff
@@ -39,7 +38,7 @@ var (
 	_ SignedData = VersionedSignedBlindedBeaconBlock{}
 	_ SignedData = VersionedSignedValidatorRegistration{}
 	_ SignedData = SignedRandao{}
-	_ SignedData = SignedBeaconCommitteeSubscription{}
+	_ SignedData = BeaconCommitteeSelection{}
```

Comment:
suggest keeping this as `SignedBeaconCommitteeSelection` to be consistent with other Signed types, we can also have `BeaconCommitteeSelection` as an unsigned type to be used in fetcher or wherever required. This way it will be more explicit which is unsigned and which is signed even internally they are same types.

Reply:
Not sure about this, as there isn't such a thing as an unsigned selection.